remove_duplicate_files_with_paren1paren.sh

```diff
@@ -23,21 +24,24 @@ then
     exit 9999 # die with error code 9999
 fi
 
-find $directory_path -type f -name '*(1).jpg' | # -delete |
+find $directory_path -type f -name '*(1).jpg' | 
+# Note: Instead of loop, could also use at end of find line: -delete |
 sort > files_deleted.txt
 
-echo 'LOOK at files_deleted.txt to VERIFY these are the files to delete.'
+echo
+echo 'Pause and LOOK at: files_deleted.txt. VERIFY these are the files to delete.'
+sleep 5s
 
 read -r -p 'If these are the files you want to delete, type Y: ' response
 response=${response,,}    # tolower
-if [[ "$response" =~ ^(Y|y|yes)$ ]]
+if [[ "$response" =~ ^(Y|y|Yes|yes)$ ]]
```

Comment:
Cool, I did not know about `"${X,,}"` as a way to lower-case the expanded value of variable `X`! Incidentally, it looks like this works only with versions of `bash` newer than the one installed by default on OS X.

If you lower-case `response`, you don't need to match against `Y` or `Yes`. Do you still want to do that?

Reply:
@kcharter What I read is that the syntax '${X,,}' works for Bash version 4+. My system has Bash version 5.0.11. So I assumed it would be all right to use & that the version I have is the default version.

WRT, Y|y|Yes|yes, I wanted to make the user pause a bit and consciously type capital Y, plus it's easier to see Y vs. y in a text screen where you can't highlight input.
But then I thought it would be nice to have the variations of Y - since I often mistype the required. I think I will leave it as Y for readability - but also keep the variations as a nice-to-have.

Reply:
@kcharter You were right. On Mac Mojave, the pre-installed Bash version is 3+.
```mac$ /bin/bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
Copyright (C) 2007 Free Software Foundation, Inc.```
What would you recommend as a general best practice & why?

Reply:
@kcharter After speaking more to you about the response of y, I have changed the code to read:
```
if [[ "$response" =~ ^(y|yes)$ ]]
```